### ISSUE TITLE
is fusion operation the same as the paper?

### ISSUE THREAD

**Issue by fortunechen** [shekkizh/FCN.tensorflow#112]
According to the original paper:
`We add a 1 × 1 convolution layer on top of pool4 to produce additional class predictions. We fuse this output with the predictions computed on top of conv7 (convolutionalized fc7) at stride 32 by adding a 2× upsampling layer and summing6 both predictions (see Figure 3). `

As my understanding, your implemtation is trying to fuse the pool4 feature directly rather than the logit result from pool4 feature. In [FCN.py](https://github.com/shekkizh/FCN.tensorflow/blob/master/FCN.py), line114:
`fuse_1 = tf.add(conv_t1, image_net["pool4"], name="fuse_1")`  

After I read a similar issue [here](https://github.com/shekkizh/FCN.tensorflow/issues/11), I still dont think it is the same as the paper. For example, if we want to get the result of FCN16 and we already constructed the FCN version of VGG, which means we have the conv6, conv7 and conv8(logits)  .

As your operation, we should do following extra operation:   
1, deconv the conv8 (extra para num: H\*W\*NUM_CLASS\*512)  
2,  fuse_1 = deconv_conv8 + pool4   
3, get result (H\*W\*512\*NUM_CLASS)

As paper, we should ：  
1,  get result from pool4(extra para num: H\*W\*NUM_CLASS\*512)   
2,  fusion the two result above

Am I wrong? Thanks !
